### ISSUE TITLE
CloudFormation Create Failed

### ISSUE THREAD

**Issue by rgstephens** [aws-solutions/qnabot-on-aws#134]
I'm receiving the following error:

```
Embedded stack arn:aws:cloudformation:us-east-1:2708018:stack/QnABot210-ExamplesStack-LDLUHWNOB826/156fe6e0-bb90-11e8-9575-50d56 
was not successfully created: The following resource(s) failed to create: 
[QuizKey, ExampleCodeVersion].
```

**Comment by rgstephens** [comment]
I suspect this is because I don't have the following permissions that I found in the index.js.

```
arn:aws:iam::${AWS::AccountId}:root
```

**Comment by awrightlilly** [comment]
I know nothing about anything. FWIW, it took 3 attempts before the stack creation worked for me. All I did was changed the name, waited a few minutes, and then initiated it again, and on the 3rd try it worked.

First two failed at different points in the process. Almost seemed like some kind of race condition where the step to create a role and assign permissions wasn't fully complete before the next step was trying to provision a service using the role? I have no idea...but just wanted to suggest you employ my finely calibrated troubleshooting technique of clicking the button again and again!
